crates/ruff_linter/src/rules/flake8_pyi/rules/redundant_none_literal.rs

```diff
@@ -130,6 +130,9 @@ pub(crate) fn redundant_none_literal<'a>(checker: &Checker, literal_expr: &'a Ex
                 literal_elements.clone(),
                 union_kind,
             )
+            .map(|fix| {
+                fix.map(|fix| fix.isolate(Checker::isolation(semantic.current_statement_id())))
+            })
```

Comment:
can you add a short comment for future readers that explains why we need to ensure isolation here?

Reply:
Yes.

Reply:
@AlexWaygood I've added the comment, can you check to see if it's good?